Abandoned AnnotationBuilder - its 'create()' method was never called

### Type of JetBrains IDE (IntelliJ, PyCharm, etc.) and version
IntelliJ IDEA 2022.3 Beta (build IU-223.7401.7)

### Operating System 
Mac OS X 13.0 (aarch64)

### TeXiFy IDEA version
0.7.24

### Description

Crash report from IDE.

### Stacktrace
```
com.intellij.diagnostic.PluginException: Abandoned AnnotationBuilder - its 'create()' method was never called: Builder{message='null', myCurrentElement={INJECTED_PSI=com.intellij.psi.impl.source.tree.injected.InjectionResult@5384223f, org.intellij.plugins.intelliLang.references.InjectedReferencesContributor$$Lambda$4842/0x00000003020b8fa0=PsiCachedValueImpl{org.intellij.plugins.intelliLang.references.InjectedReferencesContributor$$Lambda$4842/0x00000003020b8fa0@70b359d6}} (class nl.hannahsten.texifyidea.psi.impl.LatexEnvironmentImpl), myCurrentAnnotator=nl.hannahsten.texifyidea.highlighting.LatexAnnotator@7178541b, severity=INFORMATION, range=(implicit)(42795,43188)} [Plugin: nl.rubensten.texifyidea]
	at com.intellij.diagnostic.PluginProblemReporterImpl.createPluginExceptionByClass(PluginProblemReporterImpl.java:23)
	at com.intellij.diagnostic.PluginException.createByClass(PluginException.java:92)
	at com.intellij.codeInsight.daemon.impl.B.assertAnnotationCreated(B.java:324)
	at com.intellij.codeInsight.daemon.impl.AnnotationHolderImpl.assertAllAnnotationsCreated(AnnotationHolderImpl.java:275)
	at com.intellij.codeInsight.daemon.impl.DefaultHighlightVisitor.analyze(DefaultHighlightVisitor.java:94)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.analyzeByVisitors(GeneralHighlightingPass.java:307)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.collectHighlights(GeneralHighlightingPass.java:274)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.collectInformationWithProgress(GeneralHighlightingPass.java:222)
	at com.intellij.codeInsight.daemon.impl.ProgressableTextEditorHighlightingPass.doCollectInformation(ProgressableTextEditorHighlightingPass.java:97)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:57)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$1(PassExecutorService.java:405)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1086)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$2(PassExecutorService.java:397)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$13(CoreProgressManager.java:591)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:666)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:622)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:590)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:60)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.doRun(PassExecutorService.java:396)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$run$0(PassExecutorService.java:372)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:174)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:182)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:370)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:181)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
Caused by: java.lang.IllegalStateException: Abandoned AnnotationBuilder - its 'create()' method was never called: Builder{message='null', myCurrentElement={INJECTED_PSI=com.intellij.psi.impl.source.tree.injected.InjectionResult@5384223f, org.intellij.plugins.intelliLang.references.InjectedReferencesContributor$$Lambda$4842/0x00000003020b8fa0=PsiCachedValueImpl{org.intellij.plugins.intelliLang.references.InjectedReferencesContributor$$Lambda$4842/0x00000003020b8fa0@70b359d6}} (class nl.hannahsten.texifyidea.psi.impl.LatexEnvironmentImpl), myCurrentAnnotator=nl.hannahsten.texifyidea.highlighting.LatexAnnotator@7178541b, severity=INFORMATION, range=(implicit)(42795,43188)}
	at com.intellij.codeInsight.daemon.impl.B.assertAnnotationCreated(B.java:323)
	... 26 more

```

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -37,6 +37,11 @@ jobs:
       matrix:
         os: [ubuntu-latest, windows-latest]
     steps:
+    - name: configure Pagefile
+      if: matrix.os == 'windows-latest'
+      uses: al-cheb/configure-pagefile-action@v1.3
+      with:
+        minimum-size: 8GB
     - uses: actions/checkout@v2
       with:
         fetch-depth: 2

--- a/gradle.properties
+++ b/gradle.properties
@@ -1,5 +1,5 @@
 # Token for releasing to Jetbrains using the Gradle intellij/publishPlugin task, for more information see the Gradle build file.
 intellijPublishToken=mytoken
-org.gradle.jvmargs=-Xmx6144m
+org.gradle.jvmargs=-Xmx9g
 # https://plugins.jetbrains.com/docs/intellij/kotlin.html#kotlin-standard-library
 kotlin.stdlib.default.dependency = false

--- a/src/nl/hannahsten/texifyidea/highlighting/LatexAnnotator.kt
+++ b/src/nl/hannahsten/texifyidea/highlighting/LatexAnnotator.kt
@@ -125,8 +125,10 @@ open class LatexAnnotator : Annotator {
                 .create()
 
             if (element.name == "\\text" || element.name == "\\intertext") {
+                // Avoid creating an Annotation without calling the create() method
+                val range = element.requiredParameters().firstOrNull() ?: continue
                 annotationHolder.newSilentAnnotation(HighlightSeverity.INFORMATION)
-                    .range(element.requiredParameters().firstOrNull() ?: continue)
+                    .range(range)
                     .textAttributes(LatexSyntaxHighlighter.MATH_NESTED_TEXT)
                     .create()
             }

